src/attr/__init__.pyi

```diff
@@ -447,6 +447,7 @@ def make_class(
 # these:
 # https://github.com/python/mypy/issues/4236
 # https://github.com/python/typing/issues/253
+# XXX: remember to fix attrs.asdict/astuple too!
```

Comment:
is this for now or for later?

Reply:
later – its referring to the changes to mypy/typing that once they're done, we need to fix both places :)

Reply:
ack, i wondered since both links point to _closed_ issues

Reply:
I hope sOmEoNe tackles it then 😇